tests/test_bucketlist_resource.py

```diff
@@ -209,4 +269,30 @@ def test_message_for_invalid_bucketlist_id(self):
         self.assertTrue(response.status_code == 404)
         self.assertTrue(
             b'cannot delete non-existent bucketlist' in response.data)
-    
\ No newline at end of file
+
+    def test_user_bucketlist_is_not_accessed_by_other_user(self):
+        self.response = self.client.post(
+            '/api/v1/auth/login', data=json.dumps(self.other_user),
+            headers=self.headers)
+        self.response_content = json.loads(self.response.data)
+        self.headers['Authorization'] = 'JWT {}'.format(
+            self.response_content['access_token'])
+        response = self.client.get(
+            '/api/v1/bucketlists', data={},
+            headers=self.headers)
+        self.assertFalse(b'bucketlist_one' in response.data)
```

Comment:
Also test the status code and actual response when user1 attempts to access user2's bucketlist